src/TranscriptRow/index.js

```diff
@@ -226,6 +227,7 @@ TranscriptRow.propTypes = {
   transcriptionDuration: PropTypes.number,
   handleDeleteItem: PropTypes.func.isRequired,
   handleEditItem: PropTypes.func.isRequired,
+  handleClick: PropTypes.func.isRequired,
```

Comment:
Ooh - should the other PropTypes also be required? Idt propTypes at this stage are the biggest issue and you've done loads of work on this already so feel free to ignore all my comments abot them